cost-analyzer/templates/aggregator-role.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostAggregator) }}
+{{- if gt (.Values.kubecostAggregator.replicas | toString | atoi) 0 }}
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ template "aggregator.serviceAccountName" . }}
+  labels:
+    {{ include "aggregator.commonLabels" . | nindent 4 }}
+rules:
+- apiGroups:
+    - ''
+  resources:
+    - "pods/log"
```

Comment:
Huh. I think we probably copied this from elsewhere. Is this permission necessary?

Reply:
you know what, even it if is needed by something in kcm, nothing we have ported over uses it. removing.